cpp/daal/src/algorithms/k_nearest_neighbors/oneapi/bf_knn_classification_predict_kernel_ucapi_impl.i

```diff
@@ -280,6 +444,39 @@ services::Status KNNClassificationPredictKernelUCAPI<algorithmFpType>::scatterSu
     return st;
 }
 
+template <typename algorithmFpType>
+services::Status KNNClassificationPredictKernelUCAPI<algorithmFpType>::scatterBothL2Norms(ExecutionContextIface & context,
+                                                                                          const UniversalBuffer & dataSumOfSquares,
+                                                                                          const UniversalBuffer & querySumOfSquares,
+                                                                                          uint32_t dataBlockRowCount, uint32_t queryBlockRowCount,
+                                                                                          UniversalBuffer & distances)
+{
+    DAAL_ITTNOTIFY_SCOPED_TASK(compute.scatterSumOfSquares);
+    DAAL_CHECK(dataBlockRowCount <= maxInt32AsUint32T, services::ErrorBufferSizeIntegerOverflow);
+
+    services::Status st;
+    auto & kernelFactory = context.getClKernelFactory();
+    DAAL_CHECK_STATUS_VAR(buildProgram(kernelFactory));
+    auto kernel = kernelFactory.getKernel("scatter_row_col", st);
+    DAAL_CHECK_STATUS_VAR(st);
+
+    DAAL_ASSERT_UNIVERSAL_BUFFER(dataSumOfSquares, algorithmFpType, dataBlockRowCount);
```

Comment:
DAAL_ASSERT_UNIVERSAL_BUFFER(querySumOfSquares, ...)

Reply:
I'll fix it

Reply:
Fixed (in a line below)